### PR TITLE
task solution added

### PR DESCRIPTION
**Pull request by xpopovdmitryx**
- [DEMO LINK](https://xpopovdmitryx.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://xpopovdmitryx.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/card.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+.card__price {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/styles/card.css:75]:
```suggestion
```

File: src/styles/card.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  border: 1px solid transparent;
+  border-radius: 5px;
+  background: #00acdc;
+  color: #fff;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```
Comment (IvanFesenko) [inline src/styles/card.css:81]: You already set font-family
```suggestion
```

File: src/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+@import url(./card.css);
+@import url(./stars.css);
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
```
Comment (IvanFesenko) [inline src/styles/style.css:6]: font-weight: 400; - is default value - remove

**Review by IvanFesenko — CHANGES_REQUESTED** [review]

File: src/styles/card.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  width: 160px;
+  margin: 32px 19px 40px;
+}
+
+.card__product-name {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
```
[the remark this answers is not in the document]
Comment (xpopovdmitryx) [reply src/styles/card.css:22]: Wonder if style.css is the right place to store variables?

**Review by agniya-i — APPROVED** [review]

**Pull request closed without merging**